src/socket.cpp

```diff
@@ -365,6 +365,8 @@ void CSocket::OnDataReceived()
         use the signal/slot mechanism (i.e. we use messages for that).
     */
 
+    CHostAddress RecHostAddr;
+
     // read block from network interface and query address of sender
     uSockAddr UdpSocketAddr;
```

Comment:
Should really be named `saReceivedFrom`?  Maybe better if both the `CHostAddr` and `uSockAddr` variables had a same-ish name with Jamulus-ish type specifier as prefix.  (I prefer the concept of "sender address" over "received from".)

Reply:
I'll have a think about that before unmarking as draft.

Reply:
Only thing is as you go down the calls, the receiving parameters are still often called `RecHostAddr` or even just `HostAddr`. Improving the consistency of variable naming conventions is a bigger project!

Reply:
Yeah - I started thinking that too.  We say "HostAddr" without the variable saying _what it is_ a lot...  I guess the same for other data types but that seems to stand out.  In some places, where the method name makes it clear why it's consuming a host address, it might just about be acceptable.